### DISCUSSION TITLE
3D-Model for qi charger?

### DISCUSSION THREAD

**Discussion by karlscholz** [bitcraze/discussions#778] [Q&A] [answered]
Hi, in the IROS 2022 demo you used some 3d-printed landing pads that align the crazyflie with the qi charger. Is the 3D-model uploaded somewhere? Would save me some time doing the same task again :)
![grafik](https://github.com/bitcraze/discussions/assets/105341601/bb97fa41-7979-4a0e-9e53-58acc455cbc7)
https://www.bitcraze.io/2022/11/the-iros-2022-demo/

**Comment by knmcguire** [comment, marked as answer]
hi! You can find the instructions to make these chargers on the the IROS 2022 event page on the bottom: https://www.bitcraze.io/about/events/iros2022/

**Reply by karlscholz** [reply]
Ah thanks a lot 😊
